src/style.css

```diff
@@ -0,0 +1,123 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  height: 406px;
+  width: 198px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.photo-Mac {
+  height: 134px;
+  width: 160px;
+  margin-top: 32px;
+}
+
+.description {
+  text-align: left;
+  padding: 40px 16px 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.product-code {
+  padding-top: 4px;
+  text-align: left;
+  line-height: 14px;
+  font-size: 10px;
+  width: 166px;
+  color: #616070;
+  font-weight: 400;
+}
+
+.rew-and-stars {
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+  padding: 16px 16px 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center center;
+  margin-right: 4px;
+}
+
+.stars__star:nth-child(-n+4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars .stars__star:last-child {
+  margin-right: 0;
+}
+
+.reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+  align-self: center;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+  padding: 24px 16px 0;
+}
+
+.price__text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.price__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.button-buy {
+  width: 166px;
+  height: 40px;
+  margin-top: 16px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.button-buy:hover {
+  width: 164px;
```

Comment:
It's redundant, you already declare width in .button selector. For fix that proble, please check info about box-sizing: border-box